service/src/main/java/uk/gov/hmcts/reform/fpl/service/removeorder/RemoveOrderService.java

```diff
@@ -63,6 +66,32 @@ public RemovableOrder getRemovedOrderByUUID(CaseData caseData, UUID removedOrder
             .orElseThrow(() -> new RemovableOrderNotFoundException(removedOrderId));
     }
 
+    public Optional<StandardDirectionOrder> getRemovedSDO(CaseData caseData, CaseData caseDataBefore) {
```

Comment:
could just be `getRemovedSDO(List<Element<StandardDirectionOrder>> currentSDOs, List<Element<StandardDirectionOrder>> previousSDOs`